shared/js/event_dispatcher.js

```diff
@@ -101,6 +101,26 @@
     },
 
     /**
+     * Registers listener function to be executed only first time when event
+     * occurs.
+     * @param {string} eventName Name of the event to listen for.
+     * @param {function} handler Handler to be executed once event occurs.
+     */
+    once: function(eventName, handler) {
+      ensureValidEventName(eventName);
```

Comment:
In fact we need only ```ensureValidHandler(handler);``` here since ```ensureAllowedEventName``` and ```ensureValidEventName``` will be performed by ```eventDispatcher.on``` - so just kept it for the consistency. What do you think?